src/index.html

```diff
@@ -4,10 +4,302 @@
+                <div class="header__nav-line"></div>
+              </a>
+            </div>
+          </nav>
+        </div>
+
+        <div class="header__title">
+          <div class="header__line"></div>
+          <h1 class="header__text">Miami<br>Condo Kings</h1>
+        </div>
+
+      </div>
+    </header>
+
+    <main class="main-content">
+      <article class="article" id="about-us">
+        <div class="about-us">
+          <h2 class="about-us__title">
+            Miami best<br>
+            Real estate <a href="https://elitemiamirealty.com/"><span class="about-us__title about-us__title--blue">here</span></a>
+          </h2>
+
+          <section class="about-us__content">
+            <h3 class="about-us__subtitle">
+              Over 50 years experience in love.
+            </h3>
+
+            <p class="about-us__text">
+              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+            <p class="about-us__text">
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+            <p class="about-us__text about-us__text--wom">
+              <strong>We will never be late!</strong>
+            </p>
+            <p class="about-us__text about-us__text--wom">
+              <strong>Guaranteed secure transaction - 100%</strong>
+            </p>
+
+          </section>
+        </div>
+      </article>
+
+      <article class="article" id="gallery">
+        <div class="article__title">
+          <h3 class="article__title-text">Gallery</h3>
+          <div class="article__title-corona"></div>
+        </div>
+
+        <div class="gallery">
+          <section class="gallery__item">
+            <div class="gallery__pictures gallery__pictures--1">
+              <img
+                class="gallery__picture"
+                src="/images/photos/1.jpg"
+                alt="Marina Palms before"
+              >
+              <img
+                class="gallery__picture"
+                src="/images/photos/2.jpg"
+                alt="Marina Palms after"
+              >
+            </div>
+            <p class="gallery__item-label" >
+              <a href="" class="gallery__item-label gallery__item-label--blue-text">
+                <strong>
+                  Marina Palms
+                </strong>
+              </a>
+               / North Miami Beach, FL 33162
+            </p>
+          </section>
+
+          <section class="gallery__item">
+            <div class="gallery__pictures gallery__pictures--2">
```

Comment:
```suggestion
            <div class="gallery__pictures">
```
Modifier `--2` is redundant here, because this block of images has a default styles for `gallery__pictures`